Applications/VideoPlayer/pl_mpeg.h

```diff
@@ -0,0 +1,4241 @@
+/*
+PL_MPEG - MPEG1 Video decoder, MP2 Audio decoder, MPEG-PS demuxer
+
+Dominic Szablewski - https://phoboslab.org
```

Comment:
I'm afraid that's not how we do things in SerenityOS 😐

If you want to contribute a LibVideoCodec (LibVideo, LibMPEG, ...) — that would be cool. But we can't just depend on a third-party library, nor ship the whole source of a third-party library inside an app.